README.md

```diff
@@ -191,13 +191,20 @@ The rest of the publications describes different component of **Crocoddyl**:
 You have a question or an issue? You may either directly open a [new issue](https://github.com/loco-3d/crocoddyl/issues) or use the mailing list <crocoddyl@laas.fr>.
 
 
+## Executive Committee
+
+**Crocoddyl** is being maintained by an executive committee which meets every two weeks to discuss the ongoing development.
```

Comment:
`developments`

Reply:
What about to write
``**Crocoddyl** is being managed by  ...``?

I think it is better to use `managed` because we are not only maintaining the code, we are also defining the roadmap of it. Crocoddyl is still an experimental library, we are constantly developing new features.

Reply:
ok